fastlane/Fastfile

```diff
@@ -13,9 +13,14 @@ platform :ios do
   lane :build_and_upload_to_testflight do
     set_up_code_signing
 
+    latest_build_number = latest_testflight_build_number(
+      app_identifier: BUNDLE_IDENTIFIER,
+      api_key: app_store_connect_api_key
+    )
+
     increment_build_number(
       xcodeproj: File.join(APP_FOLDER, 'App.xcodeproj'),
-      build_number: latest_testflight_build_number + 1
+      build_number: latest_build_number + 1
```

Comment:
In one of our apps that are not open source, we've been using `BUILDKITE_BUILD_NUMBER` here. 

- Downside: Implicit dependency on Buildkite
- Upside: Makes it easier to find which CI job (thus, commit) created the TestFlight build
- Upside: Removes an API call to the far from reliable ASC

Reply:
>  our apps that are not open source

Is it a risk or downside to use that for Blocknotes because it's open source?

Reply:
Sorry I made a typo. It should have been "In one of our app that **is** not open source." The only reason I mentioned it was to motivate why I didn't share a link to it.